node/src/effect/requests.rs

```diff
@@ -246,14 +246,14 @@ pub enum StorageRequest {
     /// Retrieve deploys with given hashes.
     GetDeploys {
         /// Hashes of deploys to be retrieved.
-        deploy_hashes: Multiple<DeployHash>,
+        deploy_hashes: Box<Multiple<DeployHash>>,
```

Comment:
As state above, `Box<Multiple<T>>` is an anti-pattern. It is basically `Box<Vec<T>>`.

Reply:
Using simply `Multiple` blows the compile time size limit

Reply:
Changed from `Box<Multiple<T>>`  to `Vec<T>`